uploadr/UploadList.csv

```diff
@@ -141,7 +146,7 @@ Microsoft.Framework.Logging,y,y
 Microsoft.Framework.Logging.Abstractions,y,y
 Microsoft.Framework.Logging.Console,y,y
 Microsoft.Framework.Logging.NLog,n,n
-Microsoft.Framework.Logging.Testing,y,y
+Microsoft.Framework.Logging.Testing,n,n
 Microsoft.Framework.Logging.TraceSource,y,y
 Microsoft.Framework.Notification,n,n
```

Comment:
We need to remove this one as well. It's not shipping part of Beta 5 per Ryan, and it doesn't even have a Release branch.